drivers/charger/CMakeLists.txt.sbs_charger

```diff
@@ -0,0 +1,6 @@
+zephyr_library()
+
+zephyr_library_sources(sbs_charger.c)
+
+zephyr_include_directories_ifdef(CONFIG_EMUL_SBS_CHARGER .)
```

Comment:
If the header file is intended for use outside this directory, shouldn't it go into `include/zephyr/drivers/charger/` instead?

Reply:
you are referring to sbs_charger.h, correct?

Reply:
Correct.

Reply:
ack